src/webgpu/api/operation/labels.spec.ts

```diff
@@ -267,6 +267,14 @@ g.test('wrappers_do_not_share_labels')
         module,
         entryPoint: 'main',
       },
+      // Specify a color attachment so we have at least one render target. Otherwise, details here
+      // are not relevant to this test.
+      fragment: {
+        targets: [{ format: 'rgba8unorm' }],
+        module: t.device.createShaderModule({
+          code: `@fragment fn main() -> @location(0) vec4f { return vec4f(0); }`,
+        }),
+      },
```

Comment:
question: I wasn't sure if there was already a helper for trivial fragment stages. Maybe I should use one or make one?

Reply:
This is good. IMO having a single-line shader is clearer than referring out to a helper, because you can see what's happening, even if it's not really relevant to the test.